dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -628,29 +632,29 @@ export const FrontSection = ({
 								editionId={editionId}
 							/>
 							{!isOnPaidContentFront &&
-								!!editionBranding &&
-								editionBranding.branding &&
+								!!frontEditionBranding &&
+								frontEditionBranding.branding &&
```

Comment:
Is it possible to extract these long conditionals to named consts that give some insight into what the resulting boolean is? This might not be entirely accurate but something like `firstFrontEditionBranding`?. Appreciate this might be difficult so will leave it with you if you think it is possible to convey the meaning succinctly :)

Reply:
Yes! Will do, thanks for the suggestion!